adds some rfps (#6)
web3garden/sunshine-grants: rfps/substrate-ipfs.md

[Web3 RFP](https://github.com/w3f/General-Grants-Program/issues/123)

We want to use [`rust-ipfs`](https://github.com/ipfs-rust/rust-ipfs) for off-chain storage. [David recommends](https://github.com/web3garden/sunshine-grants/issues/4) the order in which the following proof patterns may be pursued:
- commit/reveal scheme
- inclusion proofs using a merkle trie
- storage proofs using the filecoin rust library

## Application Requirements

> There is an ongoing [Web3 RFP](https://github.com/w3f/General-Grants-Program/issues/123), but the title implies this is a module task. It is probably a bit more involved in for the node's client because the node will need to make a commitment to IPFS according to the runtime logic...the runtime itself does not call IPFS...

Author: ```suggestion
There is an ongoing [Web3 RFP](https://github.com/w3f/General-Grants-Program/issues/123), but the title implies this is a module task. It is likely more work on the node's client because the node will need to make a commitment to IPFS according to the runtime logic (the runtime itself does not call IPFS)
```
